packages/code/src/Code/Code.tsx

```diff
@@ -219,6 +258,19 @@ function Code({
               {...popoverProps}
             />
           )}
+
+          {showExpandButton && (
+            <button
```

Comment:
nit: you could break this out into a new component.

Reply:
I considered this, but really there wasn't much logic going on here so it didn't seem very helpful. If we decide to rotate the chevron, there may be more that will justify this though. I'll update shortly when I have more thoughts 🙂